Index: CMakeLists.txt
===================================================================
--- CMakeLists.txt	(revision 855877)
+++ CMakeLists.txt	(working copy)
@@ -4,6 +5,9 @@
 
 FIND_PACKAGE(ZLIB REQUIRED)
 
+FIND_PACKAGE(QCA2 REQUIRED)
+macro_log_feature(QCA2_FOUND "QCA2" "Qt Cryptographic Architecture" "http://delta.affinix.com/qca" FALSE "2.0.0" "Needed to build Kopete's GroupWise and Jabberprotocols.")
+
 include_directories( 
 ${CMAKE_CURRENT_SOURCE_DIR}/cutestuff/ 
 ${CMAKE_CURRENT_SOURCE_DIR}/cutestuff/legacy/ 
@@ -66,7 +70,7 @@
     ${iris_SRCS} 
 )
 
-kde4_moc_headers(iris_kopete 
+kde4_moc_headers(iris_ksirk 
 #	cutestuff/legacy/ndns.h
 #	cutestuff/legacy/safedelete.h
 #	cutestuff/legacy/servsock.h
@@ -98,23 +102,23 @@
 #	iris/xmpp-im/xmpp_discoinfotask.h
 )
 
-kde4_add_library(iris_kopete SHARED ${libiris_SRCS})
+kde4_add_library(iris_ksirk SHARED ${libiris_SRCS})
 
-set(iris_kopete_libs ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTNETWORK_LIBRARY} ${QT_QTXML_LIBRARY}
-                     ${IDN_LIBRARY} ${QCA2_LIBRARIES} ${ZLIB_LIBRARIES})
+set(iris_ksirk_libs ${QT_QTCORE_LIBRARY} ${QT_QTGUI_LIBRARY} ${QT_QTNETWORK_LIBRARY} ${QT_QTXML_LIBRARY}
+                     ${IDN_LIBRARY} ${QCA2_LIBRARIES} ${ZLIB_LIBRARIES} -lidn)
 if(WIN32)
-  set(iris_kopete_libs ${iris_kopete_libs} ws2_32)
+  set(iris_ksirk_libs ${iris_ksirk_libs} ws2_32)
 endif(WIN32)
 
 if(HAVE_NSL_LIBRARY)
   # This is probably Solaris, and libiris needs to link
   # to libnsl for gethostbyname
-  set(iris_kopete_libs ${iris_kopete_libs} nsl)
+  set(iris_ksirk_libs ${iris_ksirk_libs} nsl)
 endif(HAVE_NSL_LIBRARY)
-target_link_libraries(iris_kopete ${iris_kopete_libs})
+target_link_libraries(iris_ksirk ${iris_ksirk_libs})
 
-set_target_properties(iris_kopete PROPERTIES VERSION 1.0.0 SOVERSION 1 )
-install(TARGETS iris_kopete ${INSTALL_TARGETS_DEFAULT_ARGS})
+set_target_properties(iris_ksirk PROPERTIES VERSION 1.0.0 SOVERSION 1 )
+install(TARGETS iris_ksirk ${INSTALL_TARGETS_DEFAULT_ARGS})
 
 ########### install files ###############
 
